app/src/screens/StackNavigator.tsx

```diff
@@ -23,26 +23,22 @@ const StackNavigator = () => {
     return <Busy />;
   }
 
-  const screens =
-    user?.status === 'fulfilled' && user.accessToken ? (
-      <>
-        <Stack.Screen
-          name="App"
-          options={{ headerShown: false }}
-          component={TabNavigator}
-        />
-        <Stack.Screen name="Call" component={ActiveCall} />
-        <Stack.Screen name="Incoming Call" component={CallInvite} />
-      </>
-    ) : (
+  const isLoggedIn = user?.status === 'fulfilled' && user.accessToken;
+  if (!isLoggedIn) {
+    return <SignIn />;
+  }
```

Comment:
Have we tested this and make sure it doesn't affect the layout? Should this be wrapped in a Stack.Screen?

Reply:
Yeah, this is pretty thoroughly tested.

Reply:
It doesn't seem to be necessary to wrap it, but there has to be some pros and cons that I'm not seeing.

Reply:
@kpchoy any concerns?

Reply:
I dont think so since no screens are actually stacked, and we just need to render a single page. If it passes the UI tests then I don't think there are any concerns.